_inc/client/setup-wizard/intro-page/index.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { imagePath } from 'constants/urls';
+
+import './style.scss';
+
+const IntroPage = props => {
+	return (
+		<div className="jp-setup-wizard-intro">
+			<img
+				width="200px"
+				height="200px"
+				src={ imagePath + '/jetpack-powering-up.svg' }
+				alt={ __( 'A jetpack site powering up' ) }
+			/>
+			<h1 className="jp-setup-wizard-header">
+				{ __( 'Set up Jetpack for better site security, performance, and more' ) }
+			</h1>
+			<p className="jp-setup-wizard-paragraph">
+				{ __(
+					'Jetpack is a cloud-powered tool built by Automattic and brought to you by Bluehost.'
```

Comment:
Do you think we could make this dynamic, maybe coming from `partnerSubsidiaryId`? That data is already fed into the React interface:
https://github.com/Automattic/jetpack/blob/ca981daa705d9e209ce2a65434a8ef517991c454/_inc/client/state/initial-state/reducer.js#L282-L291

If we had no partner code provided, we'd have a generic sentence. If we had a partner code there, we could then use the Partner name here.

Reply:
I had some conversation in Slack about this and there are a number of possible solutions that need more exploration. I went ahead and removed this reference to Bluehost and I will make a PR later dedicated to customizing the content according to the partner.